rules/dead-code/tests/Rector/Class_/RemoveEmptyAbstractClassRector/Fixture/multi_extends.php.inc

```diff
@@ -36,7 +36,7 @@ abstract class AnotherAbstractClassMulti
      }
 }
 
-class MultiExtends extends AnotherAbstractClassMulti
+class MultiExtends extends \Rector\DeadCode\Tests\Rector\Class_\RemoveEmptyAbstractClassRector\Fixture\AnotherAbstractClassMulti
```

Comment:
By convention any changed class must be in FQN format, so the output is consistent